_docs/2.0.0/usage/config.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+title: Usage - Configuration
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+redirect_from: /docs/usage/config
+---
+
+Configuration
+============
+
+<!-- toc -->
+- [Configuration](#configuration)
+  - [Intro](#intro) 
+  - [General options](#general-options)
+<!-- tocstop -->
+
+Intro
+-----------
+
+This page describes the usage of configuration of _Standardization_ and _Conformance_.
+There are number of default options that 
+[Project's readme](https://github.com/AbsaOSS/enceladus/blob/develop/README.md) documents
+this page describes the configuration values stored in `spark-jobs`'s `application.conf` (or its
```

Comment:
The sentence looks like a (unwanted) merge of two sentences. Split or reformulate please.

Reply:
Split, thanks.